source/ContentScript/index.ts

```
@@ -1,3 +1,681 @@
+              startDate: formData.get(`workStartDate-${work.id}`) || '',
+              endDate: formData.get(`workEndDate-${work.id}`) || '',
+              workLocation: formData.get(`workLocation-${work.id}`) || '',
+              jobDescription: formData.get(`jobDescription-${work.id}`) || ''
+            });
+          });
+          // Handle resume upload
+          let resume = profile.resume || '';
+          let resumeUploadDate = profile.resumeUploadDate || '';
+          const fileInput = (form.querySelector('input[name="resume"]') as HTMLInputElement);
+          if (fileInput && fileInput.files && fileInput.files[0]) {
+            const file = fileInput.files[0];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)